test/unit/main_test.ts

```diff
@@ -79,19 +79,31 @@ describe('Recognizes invalid TypeScript inputs', () => {
 describe('Visitor pattern', () => {
   it('renames identifiers of property declarations/assignments', () => {
     expectTranslate('var foo = { bar: { baz: 12; } }; foo.bar.baz;')
-      .to.equal('var foo = { $: { _: 12; } }; foo.$._;');
+        .to.equal('var foo = { $: { _: 12; } }; foo.$._;');
```

Comment:
I think +4 is the correct indentation too. Make sure it consistent across this test file.

We should use clang-format to enforce formatting at some point in the future.